src/js/ace/theme-jsoneditor.js

```diff
@@ -102,7 +102,7 @@ color: #96DC5F
 color: darkorange
 }
 .ace-jsoneditor .ace_constant.ace_numeric {
-color: red
+color: rgb(0, 0, 205)
```

Comment:
Since all colors are using HEX code, it maybe good for consistency to also enter the color for numbers as HEX (`#ee422e`)

Reply:
I borrowed the `ace editor` the color of the number.